docs/apiSpec.yml

```diff
@@ -212,6 +212,40 @@ paths:
             type: integer
             example: 1
 
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                properties:
+                  graduationDate:
+                    type: string
+                    format: date
+                    example: "2026-06-30"
```

Comment:
The endpoint returns this date in full iso 8601 (e.g. `2026-06-30T00:00:00.000Z`). We should either make it not do that, or update this accordingly